src/utils/DockerUtil.js

```diff
@@ -280,6 +296,7 @@ export default {
       data.Mounts = data.Mounts || existingData.Mounts;
 
       var fullData = _.extend(existingData, data);
+      console.log(fullData);
```

Comment:
This should probably be removed, but everything else looks good - I'll merge finally!